src/EA.Weee.RequestHandlers/Mappings/EvidenceNoteDataMapBase.cs

```diff
@@ -28,6 +28,12 @@ public T MapCommonProperties(Note note)
                         .OrderByDescending(n => n.ChangedDate).FirstOrDefault())
                     ?.Reason
                     : null, // Note: do not change the order of update between Returned and Rejected as the last update wins
+                VoidedReason = note.Status.Equals(EA.Weee.Domain.Evidence.NoteStatus.Void)
```

Comment:
@GuidoWorkDev Need to update the unit tests relating to TransferEvidenceNoteMap, EvidenceNoteDataMap mappers to test the voided reason and voided date